tensorflow_addons/activations/rrelu.py

```diff
@@ -44,19 +44,28 @@ def rrelu(
         training: `bool`, indicating whether the `call`
         is meant for training or inference.
         seed: `int`, this sets the operation-level seed.
+        gs: A `Generator`. Default value is tf.random.get_global_generator().
```

Comment:
We should change this line, it's not accurate anymore.

Also we should change the argument name to `random_number_generator` or `rng` whatever you prefer. It's more common to use this name.